.github/workflows/actionlint.yaml

```diff
@@ -0,0 +1,23 @@
+name: Actionlint
+
+on:
+  pull_request:
+  push:
+    branches: main
+  # allow "manual" triggering from automatic PRs
+  workflow_dispatch:
```

Comment:
I'm not sure I understand this comment - does that apply for this repo?

Reply:
Indeed sorry, this is coming from another repo of mine. I removed the comment :+1: